lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1086,6 +1157,44 @@ paths:
         default:
           $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
 
+  /lte/{network_id}/gateways/{gateway_id}/cellular/pooling:
```

Comment:
GET/PUT are the common endpoints for gateway configs (see /cellular method). Even better we get the handlers for free via `handlers.GetPartialGatewayHandlers()`. I agree that these methods could but useful, but I think it's okay for now to wait on this. If it will simplify the NMS code for gateway pools, then I can add this in the future.